### PR TITLE
Export FireBat table to CSV

### PR DESCRIPTION
**Pull request by andrea-williams**
This PR only addresses the "CSV export table button" portion of the ticket. Copy/paste bugfix to come in another PR.

Test it out here: https://wps-pr-1271.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=239&f=c7&c=NaN&w=NaN,s=266&f=c5&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=322&f=o1b&c=90&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=1108&f=c5&c=NaN&w=NaN,s=280&f=c7&c=NaN&w=NaN,s=306&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=1055&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=286&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN

Sample CSV exported from FireBat:
[FireBAT_2021-08-30 (5).csv](https://github.com/bcgov/wps/files/7079679/FireBAT_2021-08-30.5.csv)

### CODE REVIEW

**Review by tessmgood — APPROVED** [review]
Rad TY

**Review by conbrad — APPROVED** [review]

File: web/cypress/support/fba-commands.ts [hunk 1]
```diff
@@ -44,46 +44,46 @@ declare module Cypress {
...
 
-Cypress.Commands.add('selectFBAStationInDropdown', (code: number | string) => {
+Cypress.Commands.add('selectFBAStationInDropdown', (code: number | string, rowId: number) => {
   if (typeof code === 'number') {
     return cy
-      .getByTestId('weather-station-dropdown-fba')
-      .get('button[title="Open"]')
-      .first()
+      .getByTestId(`weather-station-dropdown-fba-${rowId}`)
+      // .get('button[title="Open"]')
```
Comment (Sybrand) [inline web/cypress/support/fba-commands.ts:51]: Is this commented out code going to be used?

**Review by Sybrand — APPROVED** [review]
Approved with comments.

**Review by Sybrand — CHANGES_REQUESTED** [review]
Uh oh - found a bug.

If you selected a date for which there is no data, export selection fails without any kind of feedback for the user.

**Comment by andrea-williams** [comment]
Exporting to CSV when a station is missing data now looks like this:
[FireBAT_2021-08-31 (2).csv](https://github.com/bcgov/wps/files/7087050/FireBAT_2021-08-31.2.csv)

**Comment by andrea-williams** [comment]
> Totally not a big deal, but did notice, that on a station without a prediction - crown fraction burned is 0%. It should probably just be empty - instead of 0.
> 
> Looks good though.

Yeah I noticed that in the CSV too, but I figured that the fix was out of scope for this ticket since the table itself outputs 0 for CFB when it should just be empty.

**Pull request merged**